roottest/root/io/evolution/pragma_read/v2/AliAODForwardHeader.cxx

```diff
@@ -209,7 +209,7 @@ AliAODForwardHeader::MakeTriggerMask(const char* what)
     else if (s.CompareTo("E")          == 0) trgMask |= kE;
     else if (s.CompareTo("NCLUSTER>0") == 0) trgMask |= kNClusterGt0;
     else
-      AliWarningGeneral("MakeTriggerMask",
+      AliWarningGeneral("MakeTriggerMask", "%s",
 			Form("Unknown trigger %s", s.Data()));
```

Comment:
I suppose this would also work and be a tad bit simpler :)
```suggestion
      AliWarningGeneral("MakeTriggerMask", "Unknown trigger %s", s.Data());
```